common-rendering/src/editorialPalette.ts

```diff
@@ -87,6 +92,44 @@ const textHeadlineDark = (format: ArticleFormat): Colour => {
 	}
 };
 
+const textKicker = (format: ArticleFormat): Colour => {
+	switch(format.theme) {
+		case ArticlePillar.News:
+			return news[400];
+		case ArticlePillar.Culture:
+			return culture[400];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[400];
+		case ArticlePillar.Sport:
+			return sport[400];
+		case ArticlePillar.Opinion:
+			return opinion[400];
+		case ArticleSpecial.Labs:
+			return labs[400];
+		case ArticleSpecial.SpecialReport:
+			return specialReport[400];
+	}
+}
+
+const textInverted = (format: ArticleFormat): Colour => {
+	switch(format.theme) {
+		case ArticlePillar.News:
+			return news[500];
+		case ArticlePillar.Culture:
+			return culture[500];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[500];
+		case ArticlePillar.Sport:
+			return sport[500];
+		case ArticlePillar.Opinion:
+			return opinion[500];
+		case ArticleSpecial.Labs:
+			return specialReport[500];
```

Comment:
Labs pieces use colours from the `specialReport` palette? I'm aware this comes from the old `themeStyles` module, I'm just surprised... 🤔 

@HarryFischer this will be the colour of links in the standfirst on Guardian labs media pieces. Is this correct? Should they be using something from the labs palette instead?

Reply:
If we want to have `labs[500]`, we will have to make a change in `source`: https://github.com/guardian/source/blob/main/packages/@guardian/source-foundations/src/palette.ts#L260-L263.